chapters/hub/hub-api.md

```diff
@@ -0,0 +1,67 @@
+# Hub API
+
+Jina Hub exposes several handy commands, namely `list`, `build` and `push` with configurable parameters. These APIs constitute the user’s experience using `jina cli` for a seamless experience with the Hub.
+
+
+## `jina hub` installation:
+`hub cli` can be installed using pip, run `pip install "jina[hub]"`
+
+## `jina hub list`:
+`hub list` interface helps the user to retrieve information related to executor images in the Hub.
+It accepts filters like:
+ `name` corresponding to image name
+ `kind` corresponding to the kind of hub image (indexer / encoder / segmenter / crafter / evaluator / ranker etc)
+ `type` indicating either pod or app
+ `keywords` for filtering on specific terms like `sklearn`,  etc.
+
+`list` could be used for listing both local as well as remote images.
+
+
+## `jina hub new`:
+Jina provides an easy to use interface for spinning up an executor or an app
+### Create a new executor
+`jina hub new --type pod`
+It will start a wizard in CLI to guide you create your first executor. The resulted file structure should look like the following:
+```
+MyAwesomeExecutor/
+├── Dockerfile
+├── manifest.yml
+├── README.md
+├── requirements.txt
+├── __init__.py
+└── tests/
+    ├── test_MyAwesomeExecutor.py
+    └── __init__.py
+```
+
+
+### Create a new app
+`jina hub new --type app`
+It will start a wizard in CLI to guide you create your app.
```

Comment:
@theUnkownName 's doc on project setup covers this already. Can we link to that instead?

Reply:
It will start a wizard in **the** CLI to guide you **in creating** your app.

Reply:
Keeping the wizard description for having complete documentation for hub as this is user's first entry point to the Hub project